AdventOfCode2021/day2_2/day2_2.py

```diff
@@ -0,0 +1,39 @@
+#!python3
+
+from typing import List
+
+def file_read(file_name) -> List[str]:
+    result: List[str] = []
+    with open(file_name) as f:
+        for line in f:
+            result.append(line)
+    return result
+
+
+def main():
+    input: List[str] = file_read("day2_2.txt")
+    position_data_list: List[str] = []
+    pos_horizontal = 0
+    depth = 0
+    aim = 0
+    
+    for i in range(0, len(input)):
+        position_data_list = input[i].split(' ')
+        key = position_data_list[0]
+        value = int(position_data_list[1])
+        if ('forward' in key):
+            pos_horizontal += value
+            depth += value * aim
+        elif ('down' in key):
+            aim += value
+        elif ('up' in key):
+            aim -= value
+        else:
+            print("Something went wrong!")
+            break
```

Comment:
I would suggest `return` here, since there's no need to output result in we failed to parse the file.